packages/relay/src/lib/eth.ts

```diff
@@ -157,9 +160,40 @@ export class EthImpl implements Eth {
     this.chain = chain;
     this.precheck = new Precheck(mirrorNodeClient, nodeClient, logger, chain);
     this.cache = cache;
+    this.registry = register;
     if (!cache) this.cache = new LRU(this.options);
   }
 
+  /* Temporary code duplication for a temporary HotFix workaround while the definitive  fix is implemented
+  * Please remove initClient workaround, or better yet, do not let this be merged on MAIN, short-lived fix intended only for 0.23.1 version */
+
+
+  private requestsPerSdkClient = 0;
+  private maxRequestsPerSdkClient = process.env.ETH_CALL_MAX_REQUEST_PER_SDK_INSTANCE || 50;
+  private hederaNetwork: string = (process.env.HEDERA_NETWORK || '{}').toLowerCase();
+  private ethCallSdkClient: SDKClient | undefined;
```

Comment:
Looks like you ended up doing it for all cases. Just name it
```suggestion
  private ethSdkClient: SDKClient | undefined;
```

Reply:
good catch, yeah, the name of the variable was defined when it was only going to be used by `eth_call` indeed. 👍 

✅  done!